src/data/cel.rs

```diff
@@ -102,8 +102,16 @@ fn decode_query_string(This(s): This<Arc<String>>, Arguments(args): Arguments) -
     for part in s.split('&') {
         let mut kv = part.split('=');
         if let (Some(key), Some(value)) = (kv.next(), kv.next().or(Some(""))) {
-            let new_v: Value = decode(value).unwrap().into_owned().into();
-            match map.entry(decode(key).unwrap().into_owned().into()) {
+            let new_v: Value = decode(value)
+                .expect("failed to decode query string value")
```

Comment:
Maybe same reasoning as the json encoding applies here, wdyt?

Reply:
Tho, here, I think I'd just use the undecoded string instead when that happens

Reply:
Done